CHANGELOG.md

```diff
@@ -15,17 +15,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 ### Fixed
+
 - [Documentation] Corrected name for `voiceFocusInsufficientResources` in documentation.
+- Allow for unsubscribeFromVolumeIndicator to unsubscribe from specific callbacks.
```

Comment:
Nit: add backticks around function names. Thank you for adding a period at the end of your sentence!

Reply:
Fixed.